app/main.py

```diff
@@ -1,3 +1,47 @@
+from __future__ import annotations
+from functools import total_ordering
+from typing import Union
+
+
+@total_ordering
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
```

Comment:
From python 3.10+ you can use `|` to add multiple type hints

```suggestion
    def __init__(self, km: int | float) -> None:
        self.km = km
```

Change in all cases